### PR TITLE
test/tekton-results: check the right number of non empty records

### PR DESCRIPTION
**Pull request by Mo3m3n**
This PR updates the tekton-results e2e test from only checking the right record name to:

- checking that records have non empty data
- checking we have the right number of records corresponding to the tested pipelines

### CODE REVIEW

**Comment by adambkaplan** [comment]
@Mo3m3n can you please update the PR description to describe why this change is needed?

File: operator/test/test.sh [hunk 1]
```diff
@@ -79,15 +79,18 @@ init() {
...
+  else
     echo -n "."
     kubectl create namespace "$NAMESPACE" >/dev/null
+    # Wait for pipelines to set up all the components
+    while [ "$(kubectl get serviceaccounts -n "$NAMESPACE" | grep -cE "^pipeline ")" != "1" ]; do
+      echo -n "."
+      sleep 2
+    done
   fi
-  # Wait for pipelines to set up all the components
```
Comment (Roming22) [inline operator/test/test.sh:86]: I would not move that while block. The fact that the namespace already exists is not a guarantee that the SA has been setup.

Comment (Mo3m3n) [reply operator/test/test.sh]: Fixed

**Review by Roming22 — APPROVED** [review]

**Comment by Mo3m3n** [comment]
@Roming22 I need to fix the tests after the last rebase

**Pull request closed without merging**